### ISSUE TITLE
Remove "Accessibility Considerations" from Banner

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/vets-design-system-documentation#2430]
## Description

Per Slack Thread, remove **Accessibility Considerations** from the [Banner page](https://design.va.gov/components/banner/#accessibility-considerations).

## Tasks
- [ ] Remove Accessibility Considerations.

## Acceptance Criteria
- [ ] Accessibility Considerations has been removed.

**Comment by caw310** [comment]
@rsmithadhoc please add more details
